### PR TITLE
Update lesson.md

### PR DESCRIPTION
**Pull request by RC-PM**
Layered on more details for the product management specific work

## What does this change?

Module:
Week(s):

### CODE REVIEW

File: docs/finalproject/week-1/lesson.md [hunk 1]
```diff
@@ -22,41 +23,55 @@ sidebar_label: Lesson
...
+   - Before we introduce ourselves let's introduce one of the tools we'll be using to collaborate during the project, [Miro](https://miro.com/). Whiteboards are a simple yet powerful tool software teams use to share ideas and solve problems - Miro is just a virtual version. It should be intuitive but if you want help there's lots of [content online](https://www.youtube.com/watch?v=pULLAEmhSho).
+   
+   - Once you're familiar with the tool one person should make a copy of [this Miro board](https://miro.com/app/board/o9J_knoA0Ag=/) for your team to work on. 
+   
+   - OK, now we're setup with let's get to know each other a little using **Exercise 1: Icebreaker** on your team's copy of the Miro board.
 
-```txt
-2. Create a public Slack channel
+
+### 2. Give your team a name!
```
Comment (textbook) [inline docs/finalproject/week-1/lesson.md:34]: Looks like the teams have already mostly got their names, they're mostly in the sign up Google Sheet

File: docs/finalproject/week-1/lesson.md [hunk 2]
```diff
@@ -22,41 +23,55 @@ sidebar_label: Lesson
...
+   - Follow **Exercise 2: Finding a name** in Miro
+   
+   - This will be your identity - use it in all communications/presentations.
+
+### 3. Create a public Slack channel
```
Comment (textbook) [inline docs/finalproject/week-1/lesson.md:39]: Most have this done too

File: docs/finalproject/week-1/lesson.md [hunk 3]
```diff
@@ -22,41 +23,55 @@ sidebar_label: Lesson
...
-1. Give your team a name!
-   - This will be your identity
-   - Use it in all communications/presentations.
-```
+### 1. Introductions
+   - Before we introduce ourselves let's introduce one of the tools we'll be using to collaborate during the project, [Miro](https://miro.com/). Whiteboards are a simple yet powerful tool software teams use to share ideas and solve problems - Miro is just a virtual version. It should be intuitive but if you want help there's lots of [content online](https://www.youtube.com/watch?v=pULLAEmhSho).
+   
+   - Once you're familiar with the tool one person should make a copy of [this Miro board](https://miro.com/app/board/o9J_knoA0Ag=/) for your team to work on. 
+   
+   - OK, now we're setup with let's get to know each other a little using **Exercise 1: Icebreaker** on your team's copy of the Miro board.
```
Comment (textbook) [inline docs/finalproject/week-1/lesson.md]:
```suggestion
   - OK, now we're set up let's get to know each other a little using **Exercise 1: Icebreaker** on your team's copy of the Miro board.
```

**Review by textbook — CHANGES_REQUESTED** [review]
One minor typo to fix - some of our teams are a bit ahead and already have names/channels, but I think this is the format we'll want other cities to use in general so I'd leave it in.

**Comment by RC-PM** [comment]
Assuming OK to close this now?

**Pull request merged**